showcase/tests/integration/components/hds/icon/index-test.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, setupOnerror } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds-icon', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it should render the component with a CSS class that matches the component name', async function (assert) {
+    await render(hbs`<Hds::Icon @name="activity" />`);
+    assert.dom('svg.hds-icon').hasClass('hds-icon');
+  });
+
+  test('it has aria-hidden set to true', async function (assert) {
+    await render(hbs`<Hds::Icon @name="activity" />`);
+    assert.dom('svg.hds-icon.hds-icon-activity').hasAria('hidden', 'true');
+  });
+  test('it renders the 16x16 icon by default', async function (assert) {
+    await render(hbs`<Hds::Icon @name="activity" />`);
+    assert
+      .dom('svg.hds-icon.hds-icon-activity')
+      .hasStyle({ height: '16px', width: '16px' })
+      .hasAttribute('width', '16')
+      .hasAttribute('height', '16');
+  });
+  test('it renders the 24x24 icon when option is set', async function (assert) {
```

Comment:
small improvement to the test description
```suggestion
  test('it renders the 24x24 icon when the "size" option is set', async function (assert) {
```